cypress/integration/search.js

```diff
@@ -0,0 +1,126 @@
+if (Cypress.env('API') !== 'guillotina') {
+  describe('Search', () => {
+    beforeEach(() => {
+      cy.autologin();
+      cy.visit('/');
+      cy.createContent({
```

Comment:
@iFlameing I think this should go into the first test since it is only used there and bleeds into the other tests when added in beforeEach.

Reply:
we are using this for the first and third test and I don't want to repeat the code. It only bleeding into the second test when we are adding A and B in front.

Reply:
@iFlameing please do not try to avoid DRY in test. Tests should be explicit and readable This is far more important than DRY.

Reply:
ok, done